docs/toc/processor.rst

```diff
@@ -0,0 +1,95 @@
+Processor
+==========
+
+A pipeline component that wraps inference model and set up inference related work.
+
+
+
+Usage
+------
+
+* wrap model by setting `self.model`
+* set up inference related behaviors
+
+
+Functions
+------------------
+
+* `initialize()`: initilize model and inference related attributes.
```

Comment:
`model` or `inference` are not the required parts of a processor. From the framework perspective, `initialize` is a step that's run when a pipeline is initialized before any data (datapacks) is processed. That's the message we are going to provide.